pkg/controller/certificaterequests/acme/acme_test.go

```diff
@@ -175,8 +185,14 @@ func TestSign(t *testing.T) {
 		t.Fatalf("failed to build order during testing: %s", err)
 	}
 
-	metaFixedClockStart := metav1.NewTime(fixedClockStart)
 	tests := map[string]testT{
+		"a CertificateRequest without an approved condition should do nothing": {
```

Comment:
Suggestion: we could also test with a `CertificateRequest` that has the approved condition explicitly set to false.
(Same with other signers).